types/jquery/index.d.ts

```diff
@@ -3487,7 +3488,7 @@ interface JQuery<TElement = HTMLElement> extends Iterable<TElement> {
      * @see \`{@link https://api.jquery.com/appendTo/ }\`
      * @since 1.0
      */
-    appendTo(target: JQuery.Selector | JQuery.htmlString | JQuery.TypeOrArray<Element> | JQuery): this;
+    appendTo(target: JQuery.Selector | JQuery.htmlString | JQuery.TypeOrArray<Node> | JQuery): this;
```

Comment:
From my [testing](https://jsfiddle.net/leonard_thieu/xpvt214o/774608/), it looks like it doesn't apply to all `Node` types. Maybe it's more appropriate to make it `Element | DocumentFragment`?

Reply:
@leonard-thieu True that. In that case, we should make sure `appendTo()` and `prependTo()` accepts `Element | DocumentFragment`. Apologies for the slow reply, had a really busy week :) I'll submit the revised PR today.